magento/magento2-login-as-customer
# 144+188

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2-login-as-customer#144: "Login as Customer" functionality should be enabled by default.
2. Fixes magento/magento2-login-as-customer#175: Fix performance degradation
3. Fixes magento/magento2-login-as-customer#188: Introduce Login as Customer "is enabled" check extensibility.
 

### Manual testing scenarios (*)

1. See magento/magento2-login-as-customer#144: "Login as Customer" functionality should be enabled by default.
2. Login as Customer functionality is working correctly when it is enabled/disabled.
3. Also this PR can affect any other "Login as Customer" functionality.


### Questions or comments


### Contribution checklist (*)
